cmd/goreleaser/internal/configure.go

```diff
@@ -193,6 +193,7 @@ func DockerImage(dist, arch, armVersion string) config.Docker {
 			label("revision", ".FullCommit"),
 			label("version", ".Version"),
 			label("source", ".GitURL"),
+			"--label=org.opencontainers.image.licenses=Apache-2.0",
```

Comment:
Why not

```suggestion
			label("org.opencontainers.image.licenses", "Apache-2.0"),
```

Reply:
Not particularly familiar with goreleaser templating and how it would interpret the resulting:
```yaml
--label=org.opencontainers.image.licenses={{Apache-2.0}}
```

Reply:
Ah, I missed that it adds the template braces. In that case, your choice is the correct one.